### PR TITLE
[bitnami/airflow] Add volumeClaimTemplates for Worker StatefulSet

### PR DESCRIPTION
**Pull request by thy-vk-dao**
**Description of the change**
Add volumeClaimTemplates to create disk volume per worker

**Benefits**
Celery workers can now use persistent disk volume to store their own data like log files

**Checklist**
- [X ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X ] Variables are documented in the values.yaml and added to the README.md using (readme-generator-for-helm)[https://github.com/bitnami-labs/readme-generator-for-helm]
- [X ] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)]

### CODE REVIEW

File: bitnami/airflow/Chart.yaml [hunk 1]
```diff
@@ -32,4 +32,4 @@ name: airflow
 sources:
   - https://github.com/bitnami/bitnami-docker-airflow
   - https://airflow.apache.org/
-version: 11.3.0
+version: 11.3.1
```
Comment (javsalgar) [inline bitnami/airflow/Chart.yaml:35]: Could you bump the minor version? It is adding a non-breaking feature

**Review by javsalgar — COMMENTED** [review]
Thanks! Minor change requested

File: bitnami/airflow/Chart.yaml (hunk 1 above)
Comment (thy-vk-dao) [reply bitnami/airflow/Chart.yaml]: ok I have bumped the minor version

**Review by javsalgar — APPROVED** [review]
LGTM! Thanks

**Pull request merged**